lstchain/pointing/pointings.py

```diff
@@ -0,0 +1,86 @@
+import os
+import numpy as np
+from ctapipe.core import Component
+from ctapipe_io_lst.containers import LSTDriveContainer
+from ctapipe.core.traits import Unicode, Int
+
+__all__ = [
+    'PointingPosition'
+]
+
+
+class PointingPosition(Component):
+    """
+      Pointion position of telescopes
+
+    """
+    drive_path = Unicode(
+        '',
+        allow_none=True,
+        help='Path to the LST drive report file'
+    ).tag(config=True)
+
+    tel_id = Int(
+        0,
+        help='id of the telescope to take drive report for'
+    ).tag(config=True)
+
+    def _read_drive_report(self):
+        """
+        Reading drive reports
+
+        Returns:
+        data:`~astropy.table.Table`
+             A table of drive reports
+
+        """
+        from astropy.io import ascii
+        self.log.info("Drive report file:", self.drive_path)
+        if self.drive_path:
+                data = ascii.read(self.drive_path)
+            # Renaming the columns, since the drive report doesn't contain
+            # these information it self
+                data['col6'].name = 'time'
+                data['col8'].name = 'Az_avg'
+                data['col13'].name = 'El_avg'
+                return data
+        else:
+            raise Exception("No drive report file found")
+
+    def cal_pointingposition(self, ev_time):
+        """
+        Calculating pointing positions by interpolation
+
+        Parameters:
+        -----------
+        time: array
+            times from events
+
+        Returns
+        ----------
+        Azimuth: array
+
+        Elevation: array
+
+        """
+        timeL = ev_time[0]
+        timeH = ev_time[-1]
```

Comment:
Since `ev_time` will be just a single value, I suggest deleting these two lines:
```
timeL = ev_time[0]
timeH = ev_time[-1]
```

Reply:
Sure. We don't need it anymore